daemon/container.go

```diff
@@ -1137,6 +1138,31 @@ func (container *Container) shouldRestart() bool {
 		(container.hostConfig.RestartPolicy.Name == "on-failure" && container.ExitCode != 0)
 }
 
+func (container *Container) mountVolumes() error {
```

Comment:
can we double check that this method is still correct? Is this code affected by what it was solved in #14432?